bijection-core/src/test/java/com/twitter/bijection/TestBijectionInJava.java

```diff
@@ -49,16 +49,20 @@ public void testBasicInjection() {
     //TODO include a cleaner way to get to the scala Bijections than Bijection$.MODULE$.
     @Test
     public void testBase64Bijection() {
-        // Note, value classes return the underlying types in Java. But Java users usually
-        // don't care much about type safety, so punting on this for now
-        Bijection<byte[], String> bytes2Base64 = Bijection$.MODULE$.bytes2Base64();
+        //https://github.com/scala/scala/pull/8127
+        //generic AnyVal signatures pre-2.12.9 incorrectly had the underlying
+        //type, while in reality it's the boxed AnyVal
+        //the cast is required only for 2.11.x
+        Bijection<byte[], Base64String> bytes2Base64 = (Bijection<byte[], Base64String>)(Object)Bijection$.MODULE$.bytes2Base64();
```

Comment:
@martijnhoekstra  I think it's ok to remove `(Bijection<byte[], GZippedBase64String>)(Object)`

Reply:
That would be nice, wouldn't it? Unfortunately, the fix isn't backported to 2.11 (and most likely won't be -- 2.11 is EOL), so this is still needed on 2.11.x

Reply:
@martijnhoekstra ahh yes right, nevermind!